src/moonstone-samples/src/AlwaysViewingPanelsWithVideoSample.js

```diff
@@ -72,7 +72,7 @@ module.exports = kind({
 			{kind: IconButton, small: false, classes: 'moon-icon-video-round-controls-style'},
 			{kind: IconButton, small: false, classes: 'moon-icon-video-round-controls-style'}
 		]},
-        {name: 'panels', kind: Panels, pattern: 'alwaysviewing', classes: 'enyo-fit', showing: false, components: [
+        {name: 'panels', kind: Panels, pattern: 'alwaysviewing', showing: false, hasCloseButton: false, components: [
```

Comment:
Same here. `ApplicationCloseButton` is not added in `alwaysviewing` pattern, so there's no effect in `hasCloseButton: false`